src/main/java/uk/gov/companieshouse/pscdataapi/repository/CompanyPscRepository.java

```diff
@@ -15,4 +14,7 @@ public interface CompanyPscRepository extends MongoRepository<PscDocument, Strin
     @Query("{'company_number' : ?0, '_id' : ?1}")
     Optional<PscDocument> getPscByCompanyNumberAndId(String companyNumber, String notificationId);
 
+    @Query("{'company_number' : ?0, 'psc_id' : ?1}")
+    Optional<PscDocument> getPscByCompanyNumberAndPscId(String companyNumber, String pscId);
```

Comment:
We should be doing the get by notificationId so we can just use  getPscByCompanyNumberAndId for everything